.github/workflows/perlcritic.yml

```diff
@@ -12,10 +12,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-
-    - name: "Run Perl::Critic" 
-      uses: Difegue/action-perlcritic@2.1.0
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
-        args: cgi/*.pl scripts/*.pl lib/ProductOpener/*.pm lib/*.pl t/*.t
+        fetch-depth: 50
+
+    - name: install cpanminus to install perlcritic
+      run: sudo apt-get install -y cpanminus
+    - name: install perlcritic
+      run: sudo cpanm Perl::Critic
+
+    - name: "Run Perl::Critic"
+      run: |
+        for i in $(find . -type f \( -name "*.t" -o -name "*.pl" -o -name "*.pm" \))
+        do
+          if result=$(perlcritic ${i})
+          then    
+                  echo $result
+          else    
+                  echo -e "Problem found with ${i}:\n${result}"
+                  exit 1
```

Comment:
The test will exit for the first file that has problems, so if multiple files have problems, we won't see it right away. (e.g. right now the tests fail for food.t, but I don't know if other files will fail once food.t is fixed)

Reply:
This is true, if I can't get the GitHub action working, I'll update this.